rts/at/integration/src/main/java/org/jboss/narayana/rest/integration/RecoveryManager.java

```diff
@@ -44,6 +45,11 @@
 
     private final Map<String, ParticipantDeserializer> deserializers = new ConcurrentHashMap<String, ParticipantDeserializer>();
 
+    /**
+     * A map of participants saved in the persistent store. &lt;Participant Id :: Uid&gt;
```

Comment:
What does `<Participant Id :: Uid>` mean. It also starts a new sentence making it more confusing.

Is it suggesting that the variable is

```java
* A map of participant ids to Uids for keeping track of which participants have been saved to the recovery store.
```

Also if you are going to make changes to this PR I would suggest using the same term for the store throughout (it currently contains three different names: persistent store, object store and recovery store)

Reply:
Agree, thanks for the note. I updated the comments.